docs/layouts/partials/docs/inject/head.html

```diff
@@ -0,0 +1,99 @@
+<!-- This is a partial template (https://gohugo.io/templates/partials/) to inject into hugo-book theme header -->
```

Comment:
Copy-paste?

Reply:
I didn't get it. Copy-paste of what?
This is a comment for readers. We didn't use partials before. This is why I decided to put this hint.

Reply:
The whole html file. Did you write it or copy-paste from somewhere? Because It looks HARD for me

Reply:
I understand